tests/e2e/utils/fileHelper.js

```diff
@@ -3,7 +3,7 @@ const path = require('path')
 const { getWebDavUrl, makeApiRequest, getAuthHeaders } = require('./apiHelper')
 const config = require('../config')
 
-const userDetails = []
+let userDetails = []
```

Comment:
you can leave it a const
```suggestion
const userDetails = []
```